src/array/zip/index.ts

```diff
@@ -1,5 +1,9 @@
 export function zip(arrays: any[][]): any[][] {
```

Comment:
Thanks for your PR! I just noticed an error I made when implementing the original `zip` method. Could you please refactor the signature from:

```ts
export function zip(arrays: any[][]): any[][]
```

into:

```ts
export function zip<T>(arrays: T[][]): T[][]
```

that way we can specify the input and return types of a `zip` function 🙂

Reply:
Thank you. The same refactor is also for the inherit function, right?

Reply:
Yes thanks!